everything_else/reproducing.m

```diff
@@ -157,6 +157,81 @@
 %% Pg. 21, PTR-BCR-2
 %% Pg. 22, PTR-BCR-3 (example appears to be the same as PTR-BCR-1, and may have to be redone)
 %% Pg. 23, PTR-BCR-4 
+
+% Eq. 72
```

Comment:
Equation numbers can change, when new questions get added or old ones get removed. Instead of calling something "Eq. 72" it's better to give a description, such as "Equation in Summary section of PTR-BCR-4". Earlier we were using equation numbers in this file, but now things are changing and we're moving away from using equation numbers in that way (and we're moving more towards using descriptions).